docs/tutorials/cmip6-recipe.ipynb

```diff
@@ -62,25 +62,25 @@
      "output_type": "stream",
```

Comment:
<p>I don't think we need to reference the earlier version. None of this is really released, so we don't have to worry about backwards compatibility.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-forge/pangeo-forge-recipes/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Great. First bullet deleted in forthcoming commit.</p>